xarray/core/variable.py

```diff
@@ -46,10 +50,17 @@
     maybe_coerce_to_str,
 )
 
-NON_NUMPY_SUPPORTED_ARRAY_TYPES = (
-    indexing.ExplicitlyIndexed,
-    pd.Index,
-)
+
+@functools.cache
+def _get_non_numpy_supported_array_types():
+    """Required instead of a global to avoid circular import errors with cubed"""
+
+    return (
+        indexing.ExplicitlyIndexed,
+        pd.Index,
+    ) + DuckArrayModule("cubed").type
```

Comment:
```suggestion
    ) + DuckArrayModule("cubed").type
```

Is this necessary? I thought array api objects pass through now?